netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -12,6 +12,51 @@
 #include "ebpf_store_helper.h"
 #include "net_ebpf_ext_sock_addr.h"
 
+#define TARGET_PROCESS_ID 1234
+#define EXPIRY_TIME 60000 // 60 seconds in ms.
+
+typedef struct _net_ebpf_ext_connection_context_key
+{
+    uint64_t transport_endpoint_handle;
+    bpf_sock_addr_t destination;
```

Comment:
why is this called destination? bpf_sock_addr_t holds both source/destination information. Please rename to sock_addr or something more meaningful.

Reply:
Even though `bpf_sock_addr_t` contains both source and destination, redirect layer only get s and saves the destination address (and port). I will actually change it to not use `bpf_sock_addr_t` and use a smaller struct, with only destination info.

Reply:
Changed this now to not use `bpf_sock_addr_t`. Instead added a new `net_ebpf_ext_connect_context_address_info_t` to use in this context key.